constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayoutBaseScope.kt

```diff
@@ -678,6 +678,506 @@ abstract class ConstraintLayoutBaseScope internal constructor(extendFrom: CLObje
         return ref
     }
 
+    /**
+     * Creates a Grid based helper that lays out its elements in a single Row.
+     *
+     * @param elements [LayoutReference]s to be laid out by the Grid helper
+     * @param verticalGap defines the gap between views in the y axis
+     * @param horizontalGap defines the gap between views in the x axis
+     * @param rowWeights defines the weight of each row
+     *        the format of input string is "w1,w2,w3, ..."
+     * @param skips defines the positions in a Grid to be skipped
+     *        the format of the input string is "index:rowxcol"
+     *        index - the index of the starting position
+     *        row - the number of rows to skip
+     *        col- the number of columns to skip
+     * @param spans defines the spanned area(s) in Grid
+     *        the format of the input string is "index:rowxcol"
+     *        index - the index of the starting position
+     *        row - the number of rows to span
+     *        col- the number of columns to span
+     * @param padding sets padding around the content
+     */
+    fun createRow(
+        vararg elements: LayoutReference,
+        verticalGap: Dp = 0.dp,
```

Comment:
From what I understand, `verticalGap` wouldn't have an effect to Rows either.

Reply:
Done. Thanks for the reminder!